src/js/components/order.js

```diff
@@ -10,6 +11,55 @@ import PanelGroup from 'react-bootstrap/lib/PanelGroup';
 import Row from 'react-bootstrap/lib/Row';
 import Table from 'react-bootstrap/lib/Table';
 
+class NumberInput extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: this.props.value != undefined ? parseInt(this.props.value) : 0
+    }
+
+    this.increase = this._increase.bind(this);
+    this.decrease = this._decrease.bind(this);
+    this.update = this._update.bind(this);
```

Comment:
private method convention: underline prefix method to tell people don't use it outside of this component
```js
this._increase = this._increase.bind(this);
this._decrease = this._decrease.bind(this);
this._update = this._update.bind(this);
```